user_sync/connector/directory_ldap.py

```diff
@@ -269,9 +269,9 @@ def iter_users(self, users_filter, extended_attributes):
             if sn_value is not None:
                 user['lastname'] = sn_value
             c_value = LDAPValueFormatter.get_attribute_value(record, six.text_type('c'))
-            source_attributes['c'] = c_value
+            source_attributes['c'] = c_value.upper() if c_value else None
```

Comment:
The source_attributes should not be uppercased, because it's supposed to show through the actual source value.